client/build.gradle.kts

```diff
@@ -0,0 +1,13 @@
+
+plugins {
+    java
+}
+
+java {
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(15))
```

Comment:
Spine 1.x supports Java 8 due to some issues related to compilation. Let's use that.